com.vladfaust.unitywakatime/package.json

```diff
@@ -0,0 +1,15 @@
+{
+    "name": "com.vladfaust.unitywakatime",
+    "displayName": "Unity WakaTime",
+    "version": "1.0.0",
+    "unity": "2018.1",
+    "description": "WakaTime plugin for Unity",
+    "keywords": [
+        "unity",
+		"wakatime",
+		"stats",
+		"tracker",
+		"programming"
+    ],
+    "category": "Unity"

```

Comment:
Is that a best category name? Maybe something like "Productivity"? :thinking:

Reply:
That's a good point too. I'm not sure there's "sanctioned" category names yet, but "Productivity" makes sense.